build-system/compile/single-pass.js

```diff
@@ -329,9 +329,7 @@ exports.getGraph = function(entryModules, config) {
     // directly and which we don't want to apply during deps finding.
     .transform(babelify, {
       compact: false,
-      plugins: [
-        require.resolve('babel-plugin-transform-es2015-modules-commonjs'),
-      ],
+      plugins: ['transform-es2015-modules-commonjs'],
```

Comment:
Right now, we use two similarly named `babel-plugin`s in two different places:

**Plugins:**
https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/package.json#L58-L59

**Call sites:**
https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/build-system/build.conf.js#L51-L55
https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/build-system/compile/single-pass.js#L330-L335

I wonder if these two plugins a) do the same thing b) can be replaced by just one of them, whichever is better / newer.

Reply:
Not sure. @prateekbh?

Reply:
they both are exact opposite of each other need to check why both of them are included

Reply:
sorry, I don't have much context here as why those are used selectively

Reply:
probably @erwinmombay is the best person to asnwer

Reply:
Thanks for checking. I know I added one of them, but I'm not sure what the other one does :)

Let's punt this to its own PR.